src/core/prototypes/marks/rect.ts

```diff
@@ -193,6 +194,16 @@ export class RectElementClass extends EmphasizableMarkClass<
           }
         )
       ),
+      manager.row(
+        "Flipping",
```

Comment:
Strings file

Reply:
Need to change all strings in the project in one time. I think we need to plan it. I will not do it partially